lib/cli/src/store.rs

```diff
@@ -313,7 +314,8 @@ impl StoreOptions {
     pub fn get_store_for_target(&self, target: Target) -> Result<(Store, CompilerType)> {
         let (compiler_config, compiler_type) = self.compiler.get_compiler_config()?;
         let engine = self.get_engine_with_compiler(target, compiler_config)?;
-        let store = Store::new(engine);
+        let tunables = BaseTunables::for_target(engine.target());
+        let store = Store::new_with_tunables(&engine, tunables);
```

Comment:
We should undo this

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3483/commits/54f0f0cbe9aa4eea1afaf2bfd29b7053b2b51325